File: Pinball App 2/Pinball App 2/practice/PracticeSettingsCopy.swift
import Foundation

let practiceLeagueImportDescription = "Select name to import Lansing Pinball League scores. Automatically imports new scores throughout the season."

func importedLeagueScoreSummary(_ count: Int) -> String {
    switch count {
    case 0:
        return "No imported league scores are currently saved."
    case 1:
        return "Remove only the 1 imported league score. Manual Practice notes and scores stay."
    default:
        return "Remove only the \(count) imported league scores. Manual Practice notes and scores stay."
    }
}

func clearImportedLeagueScoresButtonTitle(_ count: Int) -> String {
    switch count {
    case 0:
        return "Clear Imported League Scores"
    case 1:
        return "Clear 1 Imported League Score"
    default:
        return "Clear \(count) Imported League Scores"
    }
}

func clearImportedLeagueScoresAlertMessage(_ count: Int) -> String {
    switch count {
    case 0:
        return "No imported league scores are currently saved."
    case 1:
        return "This removes the 1 imported league score and matching journal rows. Manual Practice entries stay."
    default:
        return "This removes the \(count) imported league scores and matching journal rows. Manual Practice entries stay."
    }
}

func clearedImportedLeagueScoresStatusMessage(_ count: Int) -> String {
    switch count {
    case 0:
        return "No imported league scores to clear."
    case 1:
        return "Cleared 1 imported league score."
    default:
        return "Cleared \(count) imported league scores."
    }
}

File: Pinball App Android/app/src/main/java/com/pillyliu/pinprofandroid/practice/PracticeSettingsCopy.kt
package com.pillyliu.pinprofandroid.practice

internal const val PRACTICE_LEAGUE_IMPORT_DESCRIPTION =
    "Select name to import Lansing Pinball League scores. Automatically imports new scores throughout the season."

internal fun importedLeagueScoreSummary(count: Int): String {
    return when (count) {
        0 -> "No imported league scores are currently saved."
        1 -> "Remove only the 1 imported league score. Manual Practice notes and scores stay."
        else -> "Remove only the $count imported league scores. Manual Practice notes and scores stay."
    }
}

internal fun clearImportedLeagueScoresButtonTitle(count: Int): String {
    return when (count) {
        0 -> "Clear Imported League Scores"
        1 -> "Clear 1 Imported League Score"
        else -> "Clear $count Imported League Scores"
    }
}

internal fun clearImportedLeagueScoresAlertMessage(count: Int): String {
    return when (count) {
        0 -> "No imported league scores are currently saved."
        1 -> "This removes the 1 imported league score and matching journal rows. Manual Practice entries stay."
        else -> "This removes the $count imported league scores and matching journal rows. Manual Practice entries stay."
    }
}

internal fun clearedImportedLeagueScoresStatusMessage(count: Int): String {
    return when (count) {
        0 -> "No imported league scores to clear."
        1 -> "Cleared 1 imported league score."
        else -> "Cleared $count imported league scores."
    }
}
